### DISCUSSION TITLE
The Hasura Community Call 🎉

### DISCUSSION THREAD

**Discussion by mehtavishwa30** [hasura/graphql-engine#7262] [General] [answered]
📢  The Hasura Community Call is back and this time we're launching lots of exciting features and will showcase some of them live—so don't miss it! 😎

✔️  Improvements with Inherited Roles
✔️  Query Tags
✔️  Improvements with Computed Fields
✔️  CLI ARM Release, & more!

Check out the agenda & [register](https://hasura.io/community/community-call/)!

**Comment by Adron** [comment, marked as answer]
Signing up @ [https://hasura.io/community/community-call/](https://hasura.io/community/community-call/)!!
